README.md

```diff
@@ -1 +1,168 @@
-# redirects
\ No newline at end of file
+# Url Redirects with GitHub
+
+_Counterpart to the [redirects-website repo](../../../redirects-website)._
+
+✏️ [How to edit](#how-to-edit)  
+💡 [Motivation](#motivation)  
+✔️ [Comparison](#comparison)  
+🧠 [How it works](#how-it-works)  
+⚙️ [Setup](#setup)
+
+## How to edit
+
+1. Add/change/remove redirect entries in one or more [`.yaml` files in the top folder](../../blob/main/redirects.yaml).
+2. Commit the changes to the `main` branch.
+3. Changes should take effect automatically within a minute or so.
+   Verify that no errors occurred in the automatic process here: [![Encode and deploy](../../actions/workflows/deploy.yaml/badge.svg)](../../actions/workflows/deploy.yaml)
+
+You can do this [directly on github.com](../../edit/main/redirects.yaml) (tip: press <kbd>.</kbd> right now), or locally with git.
+
+## Motivation
+
+You've likely heard of a service called [Bitly](https://bitly.com/).
+It allows you to convert a long link like `some-website.com/a-long-url?search=a-bunch-of-characters` into a shorter one like `bit.ly/98K8eH`.
+When someone visits the shorter link, Bitly automatically "redirects" them to the longer one.
+You can think of it like a shortcut.
+
+Bitly also offers several other features that you probably want:
+
+- You can customize the text after the `/`, giving you a url that a human could actually remember and type in manually, like `bit.ly/MyCoolLink` (they call these "back-halves").
+- You can set up a custom domain to brand your links the way you want, giving you an even nicer and easier-to-use link, like `my-website.com/MyCoolLink`.
+- You can see how many and what kinds of people have used the link over time (i.e. analytics).
+
+This all sounds great, so what's the problem?
+Well namely: _Bitly [hides or limits a lot of this functionality behind a paywall](https://bitly.com/pages/pricing)_, and their free plan has become more and more limited over time.
```

Comment:
For example, bitly charges over $40,000 per year for the feature of changing the destination link.

Reply:
And with Bitly, each person has their own account, so unless you share passwords (eww), it is difficult for teams to collaborate on a single set of redirection links.

Reply:
Unless you're willing to spend thousands of dollars a year, there's no concept of a team account https://bitly.com/pages/pricing

No need to cite specific costs, but you could point to their pricing page

Reply:
I am linking to Bitly's pricing page, yes.

So it looks like they do have "teams" but only with the enterprise plan, yeah:

> ENTERPRISE
Drive brand loyalty, scale multichannel initiatives and collaborate across teams with a solution tailored to your needs.
Get a Quote
10,000+ branded links/mo.
Everything in Premium, plus:
Multiple user seats and group permissions

I'll word it carefully to reflect this.